src/main/kotlin/blackjack/domain/Players.kt

```diff
@@ -0,0 +1,7 @@
+package blackjack.domain
+
+data class Players(
+    val names: List<String>
+) {
+    var players: List<Player> = names.map{ Player(it) }
```

Comment:
```suggestion
    val players: List<Player> = names.map { Player(it) }
```

불변(val)의 사용 검토와 스페이스를 하나 넣어주시면 좋겠네요~ ktlint를 pre-commit hook 등에 넣어두거나 하는 형태로 코드 컨벤션을 맞춰주시면 좋겠습니다.